charts/thoras/templates/api-server/webhooks.yaml

```diff
@@ -93,3 +93,27 @@ webhooks:
     timeoutSeconds: 5
     admissionReviewVersions:
       - v1
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: "validation.thoras.ai"
+webhooks:
+  - name: "ast-validation.thoras.ai"
+    rules:
+    - apiGroups:   ["thoras.ai"]
+      apiVersions: ["v1"]
+      operations:  ["CREATE", "UPDATE"]
+      resources:   ["aiscaletargets"]
+      scope:       "Namespaced"
+    clientConfig:
+      service:
+        namespace: {{ .Release.Namespace }}
+        name: thoras-api-server
+        path: /validate
+      caBundle: {{ $secretCaCert | quote }}
+    sideEffects: None
+    failurePolicy: 'Ignore'
```

Comment:
Just to confirm, we want to allow them to make an incorrect edit to an AST if our validator fails for some reason?

Reply:
This is worth discussing as a group on Monday! We take this approach for all other webhooks but this might be a special case where we don't want them to continue with changes if we cannot verify them. 

On the other hand, this could stop them from applying critical fixes when modifying an AST.